src/pioneer-p3dx/p3dx_dpl/src/motor_controler.cpp

```diff
@@ -40,10 +44,9 @@ void callback_val(const geometry_msgs::Twist::ConstPtr &msg) {
 }
 
 int main(int argc, char **argv) {
-  std::string pub_right_ = "hal/rightMotor/setVel";
-  std::string pub_left_ = "hal/leftMotor/setVel";
+  std::string pub_right_ = "/hal/rightMotor/setVel";
+  std::string pub_left_ = "/hal/leftMotor/setVel";
```

Comment:
Sry I have worked on not rebased branch and I have not noticed that you have reworked hierarchy.  I will fix it in next pull request.